### ISSUE TITLE
Scopes.get() - getting 'utf8' is not a supported encoding name error

### ISSUE THREAD

**Issue by andrewchilcott** [sendgrid/sendgrid-csharp#367]
#### Issue Summary

When I call sg.client.scopes.get() I receive an error message containing 'utf8' is not a supported encoding name.

#### Steps to Reproduce
This is the code that I am using:
```
        public static async Task<ScopeList> getScopes()
        {
            ScopeList result = new ScopeList();
            dynamic response = await Shared.sg.client.scopes.get();
            if (response.StatusCode.Equals(HttpStatusCode.OK))
            {
                result = JsonConvert.DeserializeObject<ScopeList>(response.Body.ReadAsStringAsync().Result);
                result.userMessage = "Success - " + result.Scopes.Count.ToString() + "  retrieved at " + DateTime.Now.ToString() + ".";
            }
            else
            {
                result.userMessage = Shared.GenerateFailureMessage(response, "scopes_get_all", "");
            }

            return result;
        }

```
Where ScopeList is a simple class that I use internally:

```
    public class ScopeList
    {
        [JsonProperty(PropertyName = "scopes")]
        public List<string> Scopes { get; set; }
        [JsonIgnore]
        public string userMessage { get; set; }
    }

```
The error occurs in the call to response.Body.ReadAsStringAsync().Result.

Response.Body.bufferedContent.Length = 2486 and the buffer appears to contain a valid array of strings ie. { "scopes": [ alerts,create , alerts..... etc - I have only decoded this far.

Any reference to utf8 should actually refer to utf-8. Is there a typo somewhere in the source code?

 #### Technical details:

* sendgrid-csharp Version: 8.0.5
* .NET Version: 4.5.2

**Comment by thinkingserious** [comment]
Hello @andrewchilcott,

Did you mistakenly close this issue or have you found a solution?

**Comment by andrewchilcott** [comment]
Elmer

I closed it because I duplicated the posting #367 and #368.

#368 remains open and still a problem I need to solve before I can go any further with this particular piece of code.

Andrew

**Comment by thinkingserious** [comment]
Got it, thanks!
